applications/RomApplication/python_scripts/structural_mechanics_static_rom_solver.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+# Import base class file
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_static_solver import StaticMechanicalSolver
+import KratosMultiphysics.RomApplication as romapp
+
+def CreateSolver(main_model_part, custom_settings):
+    return ROMSolver(main_model_part, custom_settings)
+
+class ROMSolver(StaticMechanicalSolver):
+    """The stationary class for ROM structural mechanics solvers.
+
+    See structural_mechanics_static_solver.py for more information.
+    """
+
+    def __init__(self, main_model_part, custom_settings):
+        super(ROMSolver, self).__init__(main_model_part, custom_settings)
+        KratosMultiphysics.Logger.PrintInfo("::[ROMSolver]:: ", "Construction finished")
+
+    #### Private functions ####
+    @classmethod
+    def GetDefaultSettings(cls):
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "rom_settings": {
+            "nodal_unknowns": [ "DISPLACEMENT_X", "DISPLACEMENT_Y"],
```

Comment:
```suggestion
            "nodal_unknowns": [ "DISPLACEMENT_X", "DISPLACEMENT_Y", "DISPLACEMENT_Z"],
```

?

Reply:
This change is only esthetic, due to the way the parameters are being validated inside the RomB&S, where the lenght of this list dictates the number of nodal unknows. Perhaps we can even leave this parameters aside, by modifying the B&S instead...